crates/hir_def/src/item_tree/pretty.rs

```diff
@@ -493,14 +493,14 @@ impl<'a> Printer<'a> {
                 w!(self, "]");
```

Comment:
I'm not sure what this pretty printer is/does so I don't know if we want to also include the identifiers here. I figured I'd keep the original functionality for now so no functional changes here.

Reply:
Pretty printer is mainly a debugging tool for us to visualize our `ItemTree` collection